public/style/bzlite.css

```diff
@@ -778,6 +778,41 @@ input[disabled] {
   border-radius: 0;
 }
 
+#search {
+  display: flex;
+  flex-direction: row;
+}
+
+#search a {
+  color: #00AACC;
+  padding: .7rem 0 .7rem 1.5rem;
+  font-style: italic;
+}
+
+#search input {
+  flex-shrink: 1;
+}
+
+#search .close {
+  flex-grow: 1;
+}
+
+#searchLink {
+  display: block;
+  background: white;
+}
+
+#searchLink span {
```

Comment:
Made it a div, its just a placer since I needed to give the background image + content an opacity but not the background color, cheers